mne/viz/topo.py

```diff
@@ -489,9 +754,25 @@ def plot_topo_image_epochs(epochs, layout=None, sigma=0.3, vmin=None,
     border : str
         matplotlib borders style to be used for each sensor plot.
     fig_facecolor : str | obj
-        The figure face color. Defaults to black.
+        The figure face color. Defaults to black. If "black", spines are
```

Comment:
so do you allow other colors as `fig_facecolor`? what happens to the spine color then? Maybe we should just allow two colors and throw an error for the rest?